modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -385,26 +388,58 @@ def value_counts(self, **kwargs):
         return self.__constructor__(new_frame, shape_hint="column")
 
     def _get_index(self):
+        """
+        Return frame's index.
+
+        Returns
+        -------
+        pandas.Index
+        """
         if self._modin_frame._has_unsupported_data:
             return default_axis_getter(0)(self)
         return self._modin_frame.index
 
     def _set_index(self, index):
+        """
+        Set new index.
+
+        Parameters
+        ----------
+        index : pandas.Index
+            A new index.
+        """
         if self._modin_frame._has_unsupported_data:
-            return default_axis_setter(0)(self, index)
-        default_axis_setter(0)(self, index)
-        # NotImplementedError: OmnisciOnRayFrame._set_index is not yet suported
-        # self._modin_frame.index = index
+            default_axis_setter(0)(self, index)
+        else:
+            default_axis_setter(0)(self, index)
+            # NotImplementedError: OmnisciOnRayFrame._set_index is not yet suported
+            # self._modin_frame.index = index
```

Comment:
This change doesn't seem to relate to the issue.

Reply:
This is required to pass doc checker. `return` in the code requires a corresponding section in docstrings.

Reply:
ok